projects/demo-playwright/tests/demo/landing.spec.ts

```diff
@@ -7,7 +7,11 @@ test.describe('Landing', () => {
     });
 
     test('take snapshot', async ({page}) => {
-        await tuiGoto(page, '/', {hideHeader: false});
+        await tuiGoto(page, '/', {
+            hideHeader: false,
+            checkStable: false,
```

Comment:
Why shouldn't we wait for stable state here ?

Reply:
fixed